Content.Server/Mind/Mind.cs

```diff
@@ -27,21 +28,54 @@ namespace Content.Server.Mind
     ///     Things such as respawning do not follow, because you're a new character.
     ///     Getting borged, cloned, turned into a catbeast, etc... will keep it following you.
     /// </remarks>
-    public sealed class Mind
+    [DataDefinition]
+    [Serializable]
+    public sealed partial class Mind
     {
         private readonly MobStateSystem _mobStateSystem = default!;
         private readonly GameTicker _gameTickerSystem = default!;
         private readonly MindSystem _mindSystem = default!;
-        [Dependency] private readonly IPlayerManager _playerManager = default!;
-        [Dependency] private readonly IEntityManager _entityManager = default!;
-        [Dependency] private readonly IAdminLogManager _adminLogger = default!;
+
+        // I am trully sorry for this.
+        [Dependency] private IPlayerManager __playerManager = default!;
```

Comment:
I don't understandm it seems that after i added the Serializable attribute to the class, dependency injection silently stopped working.